docs/reference/modules/page.md

```diff
@@ -0,0 +1,379 @@
+# `@apostrophecms/page`
+
+**Alias:** `apos.page`
+
+This module provides the majority of functionality for serving and generally working with [pages](/reference/glossary.md#page) in Apostrophe.
+
+For creating page types, see the `@apostrophecms/page-type` module instead.
+
+## Options
```

Comment:
The options table and individual descriptions are copied almost verbatim from the existing module options page. https://v3.docs.apostrophecms.org/reference/module-api/module-options.html